wagtail/admin/views/pages/revisions.py

```diff
@@ -39,6 +40,19 @@ def revisions_revert(request, page_id, revision_id):
         instance=revision_page, request=request, form=form
     )
 
+    if revision_page.live and revision_page.has_unpublished_changes:
+        # Page status needs to present the version of the page containing the correct live URL
+        page_for_status = page.specific
+    else:
+        page_for_status = revision_page
+
+    action_menu = PageActionMenu(request, view="revisions_revert", page=page)
+    side_panels = PageSidePanels(
+        request,
+        page_for_status,
+        comments_enabled=form.show_comments_toggle,
+    )
```

Comment:
Don't know if it's relevant for revisions, maybe we should hide it in the template instead? Without this, there's still some missing files errors. I'm not very familiar with the new page editor stuff yet, so I just followed what's needed in the edit page's `"media"` context.